specification/eventgrid/data-plane/Microsoft.Media/stable/2018-01-01/MediaServices.json

```diff
@@ -451,7 +451,7 @@
           "additionalProperties": {
             "type": "string"
           },
-          "description": "Gets the Job correlation data."
+          "description": "Gets the Job correlation data. Schema of the data property of an EventGridEvent for an Microsoft event."
```

Comment:
I don't think this description should change?

Reply:
Yep, good catch. Updated the PR. I have also updated all event descriptions to use "a Microsoft... event.".